secant/Features/Home/Views/HomeView.swift

```diff
@@ -0,0 +1,116 @@
+import SwiftUI
+import ComposableArchitecture
+
+struct HomeView: View {
+    let store: Store<HomeState, HomeAction>
+
+    var body: some View {
+        WithViewStore(store) { viewStore in
+            VStack {
+                Button(
+                    action: { viewStore.toggleShowingHistory() },
+                    label: { Text(viewStore.historyToggleString()) }
+                )
+                    .primaryButtonStyle
+                    .frame(height: 50)
+
+                Button(
+                    action: { viewStore.toggleSelectedTransaction() },
+                    label: { Text("Toggle Selected Transaction") }
+                )
+                    .primaryButtonStyle
+                    .frame(height: 50)
+                Spacer()
```

Comment:
Sure!, do you mean between the buttons?

Reply:
There is one inbetween the buttons, I mean above the Spacer

Reply:
Yeah we've been trying to put a space around each view in code so that the structure shows better.  We actually added this to our style guide I believe.  The first and last don't need one on the top and bottom but a space in-between them helps with readability.

Reply:
Done in 86d77fd61911bc65ebd2d72e49cb453a00b9cb3c